### PR TITLE
Update concepts.md

### PR DESCRIPTION
**Pull request by theo-cox**
[m]: updated with revised building blocks

### CODE REVIEW

**Comment by rufuspollock** [comment]
@theo-cox is there a reason this isn't merged? If you want my review put me as a reviewer and @ me ...

File: content/notes/concepts.md [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+Business models are composed of **building blocks** which can be assembled under **constraints **to make a **limited number of business model types**. Learning these types–and the list of blocks and the rules for assembly–allow you to rapidly design business models that are successful for chosen outcomes.
 
 Before outlining options for business model types, we must define and outline terminology. 
 
-**Digital goods** are all those economic goods which are made up of data or information. These can range from software to datasets to music and even pieces of intellectual property such as pharmaceutical patents. These goods can also be known as information goods, information products and similar terms. 
+**Digital goods **are all those economic goods which are made up of data or information. These can range from software to datasets to music and even pieces of intellectual property such as pharmaceutical patents. These goods can also be known as information goods, information products and similar terms.
```
Comment (rufuspollock) [inline content/notes/concepts.md:7]: That is a formatting bug

File: content/notes/concepts.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
 # Concept Definitions
 
-Business models are composed of **building blocks** which can be assembled under **constraints** to make a **limited number of business model types**. Learning these types–and the list of blocks and the rules for assembly–allow you to rapidly design business models that are successful for chosen outcomes.
+Business models are composed of **building blocks** which can be assembled under **constraints **to make a **limited number of business model types**. Learning these types–and the list of blocks and the rules for assembly–allow you to rapidly design business models that are successful for chosen outcomes.
```
Comment (rufuspollock) [inline content/notes/concepts.md:3]: @theo-cox formatting bug ... should be 

```
**constraints** ...
```

File: content/notes/concepts.md [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
 
 **Building blocks** refer to the different components a business model needs to function. The core building blocks are as follows:
 
 
 
-* **Revenue source** - where money comes from
-* **Compensation mechanism** - how contributors are compensated for their work
-* **Governance model**- how decisions are made
-* **Rights framework** - how rights for access and use are defined 
+* **Revenue model** - how you earn revenue
```
Comment (rufuspollock) [inline content/notes/concepts.md:15]: Nice ... much clearer

File: content/notes/concepts.md [hunk 4]
```diff
@@ -1,32 +1,33 @@
...
 
-There are a number of different options for each building block. There are **constraints** on how different building block options can be combined. This limits the number of combinations which are possible, as choices around one building block can limit others. For example, not every revenue source is compatible with every rights framework. We will elaborate more on constraints below.
 
 The upshot of these constraints is that there are a limited number of **business model types** which can be created. Types are just generalized combinations of building blocks, and form the basis of any specific business model which is then created in reality. By analogy “seeded wholemeal” is a generalized type of bread, although each major supermarket may create their own real world instantiation of this type.
 
 Different model types will have different **features.** These are specific characteristics of how the business model functions. The major features we will consider are:
 
 
+
 * **Level of access** - how widely accessible the business model makes data
```
Comment (rufuspollock) [inline content/notes/concepts.md:28]: Minor style point: usually you'd have `:` here no `-` e.g.

```
**Level of access**: how widely accessible the business model makes data
```

File: content/notes/concepts.md [hunk 5]
```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
 
 The diagram below shows the different options available for each building block. We will describe these below. Note that even these options are themselves broad umbrellas and can themselves branch into further variations when developed further.
 
-![Building Blocks (1)](https://user-images.githubusercontent.com/72795023/228890509-72ea53ef-b285-4af8-bb70-0fd720525adb.svg)
 
+![building blocks](https://user-images.githubusercontent.com/72795023/232063665-80cbe3c0-5a21-42e4-95d5-e59258f869eb.svg)
```
Comment (rufuspollock) [inline content/notes/concepts.md:40]: It would be preferable to put svg in the repo - but we can fix this later.

File: content/notes/concepts.md [hunk 6]
```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
...
-## Revenue Source
 
-This building block determines where the money comes from. There are four broad options here:
 
+## Revenue model
 
+This building block determines how the business model earns the revenue to support it. There are four broad options here, while they can be mixed together:
 
-* **Grants/Subsidies** - revenue comes exclusively from grants or subsidies, usually from public bodies or philanthropists but also possible other sources such as sector associations .
+* **Grants/Subsidies** - revenue comes exclusively from grants or subsidies, usually from public bodies or philanthropists but also possible other sources such as sector associations. These can be **_conditional or unconditional._**
```
Comment (rufuspollock) [inline content/notes/concepts.md:48]: Italics and bold is probably not needed for conditional and unconditional

File: content/notes/concepts.md [hunk 7]
```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
...
-
-## Hard Constraints
-
-There is only one genuine hard constraint on combining building blocks. This is that **an Open access rights framework is incompatible with any revenue model relying on user fees, either partially or entirely.** Definitionally, if anyone has to pay to access data, then that data is not Open. 
-
-
-## Soft Constraints 
-
-There are far more soft constraints, however. These are as follows:
+There are a number of constraints on how you combine model building blocks. We will explore these in detail in later work, but for now the two most obvious ones are as follows:
```
Comment (rufuspollock) [inline content/notes/concepts.md:92]: Lot of white space here ...

File: content/notes/concepts.md [hunk 8]
```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
...
-* None/donations is an infeasible revenue source for an entity or project which predominantly uses fixed or conditional payments as a compensation mechanism.
-    * **Reason**: it will be hard to gather enough donations to make sufficient monetary payments of these kinds.
-* As a rule of thumb, the less guaranteed compensation offered under the compensation mechanism (moving to the right on the diagram, excluding mixed models) the more democratic one should make the governance model (moving to the right on the diagram).
-    * **Reason**: participation in governance can act as an added incentive to contributors in the place of money. Also, where compensation depends on the success of the entity or project as in the conditional payment and (to a lesser extent) complementary goods/services compensation mechanisms, contributors will likely want to be able to influence this outcome.
+* You cannot have a paid supplier remuneration model without some form of revenue model. This model also needs to generate sufficient revenue to pay enough suppliers at the requisite level. 
+* You cannot have a fully Open access business model and have user fees of any kind.
```
Comment (rufuspollock) [inline content/notes/concepts.md:96]: Strictly, you could with dual licensing so maybe phrase as ...

```
A fully open access business model by definition means that user fees must be zero, at least for direct access 
```

File: content/notes/concepts.md [hunk 9]
```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
 
 
 * **Access**: optimizing for social welfare will mean selecting a business model which prioritizes high levels of access. By contrast, business models which use closed rights frameworks may enable you to make more money (although this is not always the case, as businesses such as Facebook demonstrate).
```
Comment (rufuspollock) [inline content/notes/concepts.md:112]: "you" here should be clarified e.g.

... may enable the *database provider* to make more money

File: content/notes/concepts.md [hunk 10]
```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
...
-* **Quality:** optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. This is a matter of cost-benefit analysis, trading off gains in quality most often against levels of provision (i.e. the same resources can get you more lower quality digital goods or fewer higher quality digital goods) By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
+* **Quality**: optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
```
Comment (rufuspollock) [inline content/notes/concepts.md:113]: Maybe, for fairness add a comment lkie ...

Obviously, business models that generate little revenue will  struggle to maintain quality.

File: content/notes/concepts.md [hunk 11]
```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
...
 * **Profitability**: different models will be differently profitable. Crucially, profitability can often seem to come into conflict with access. Optimizing for money making via profit may mean the use of fees and other mechanisms which reduce access and thus social welfare. On the flip side, the most obvious route to high access and thus social welfare–free universal public provision–is innately unprofitable by definition.
```
Comment (rufuspollock) [inline content/notes/concepts.md:114]: Not just unprofitable but may lack revenue ...

Strictly, from a social welfare point of view profitability isn't really a feature at all ... you'd only have access and quality ... (i wonder therefore if we drop profitability for now ...?)

File: content/notes/concepts.md [hunk 12]
```diff
@@ -1,32 +1,33 @@
...
 
-There are a number of different options for each building block. There are **constraints** on how different building block options can be combined. This limits the number of combinations which are possible, as choices around one building block can limit others. For example, not every revenue source is compatible with every rights framework. We will elaborate more on constraints below.
 
 The upshot of these constraints is that there are a limited number of **business model types** which can be created. Types are just generalized combinations of building blocks, and form the basis of any specific business model which is then created in reality. By analogy “seeded wholemeal” is a generalized type of bread, although each major supermarket may create their own real world instantiation of this type.
 
 Different model types will have different **features.** These are specific characteristics of how the business model functions. The major features we will consider are:
```
Comment (rufuspollock) [inline content/notes/concepts.md:24]: 0️⃣ formatting in that fullstop should be outside the bold.

File: content/notes/concepts.md [hunk 13]
```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
 
 The diagram below shows the different options available for each building block. We will describe these below. Note that even these options are themselves broad umbrellas and can themselves branch into further variations when developed further.
```
Comment (rufuspollock) [inline content/notes/concepts.md:37]: broad umbrellas => broad categories

Could tweak that second sentence to something like:

Note that these options are categories and each one can be varied and tweaked in a variety of ways. Our aim here is to give a broad outline and distinguish the main categories from which one begins to assemble a business model. Think of it like a car: you choose the make and model and the broad features and then at the end you choose details like the specific color or trim or whether it has a sunroof or not.

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Made some suggestions for improvements. Can obviously merge as is and then make the changes or make the changes directly and then merge.

**Comment by theo-cox** [comment]
@rufuspollock all suggestions now merged

**Pull request merged**